.github/workflows/pull_requests.yml

```diff
@@ -160,7 +160,7 @@ jobs:
         run: make deploy-examples-update
       - name: Deploy update for example with configuration in CRD
```

Comment:
```suggestion
      - name: Deploy update for example with configuration in CR
```

Reply:
Changed in https://github.com/SumoLogic/tailing-sidecar/pull/162/commits/38f885df52f94adcc40a1f3a3a0a1daad46697e9